### PR TITLE
Hacker Portal Page

### PR DESCRIPTION
**Pull request by hunterchen7**
image

### CODE REVIEW

File: src/components/live/mentors.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Mentors = () => {
+  return <div>mentors</div>;
```
Comment (LordExodius) [inline src/components/live/mentors.tsx:2]: inconsistent casing :( 👎

File: src/pages/login.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default function Login() {
         });
         return;
       }
-      void router.push("/dashboard");
+      void router.push("/live");
```
Comment (LordExodius) [inline src/pages/login.tsx:34]: can we keep this as redirect to dashboard until live site is ready

**Review by LordExodius — CHANGES_REQUESTED** [review]
lil change

File: src/components/live/mentors.tsx (hunk 1 above)
Comment (hunterchen7) [reply src/components/live/mentors.tsx:2]: fixewd

File: src/pages/login.tsx (hunk 2 above)
Comment (hunterchen7) [reply src/pages/login.tsx]: fixed

**Review by LordExodius — APPROVED** [review]
epic lgtm 👍

**Pull request merged**